frontend/src/console/components/Hero/Hero.tsx

```diff
@@ -160,6 +181,11 @@ export class HeroComponent extends React.PureComponent<Props, State> {
         homepageOverlayHtml
       );
     }
+    if (this.state.title === '' || this.state.subtitle === '') {
```

Comment:
We should `trim()` these values, because if we input whitespace here it won't be sent to backend and the hero preview will not match this field's value.

Reply:
Yeah, added the trim method around the strings to remove the whitespaces, will remember this too from next time :+1: